src/library/Graphs/PayoutLine.tsx

```diff
@@ -103,8 +104,13 @@ export const PayoutLine = (props: PayoutLineProps) => {
         display: false,
       },
       title: {
-        display: false,
-        text: `${network.unit} Payouts`,
+        position: 'left' as const,
+        display: true,
+        text: `${average} Day Average`,
+        padding: 0,
+        font: {
+          size: 10,
+        },
```

Comment:
probably could use some help on styling, but i think it looks okay atm. perhaps would want to remove some padding from the left